src/app/parcels/ActionBar/Statuses.tsx

```diff
@@ -18,22 +19,25 @@ export const statusNames = [
     "Called and Confirmed",
     "Called and No Response",
     "Shopping List Downloaded",
+    "Day Overview Downloaded",
     "Ready to Dispatch",
     "Received by Centre",
     "Collection Failed",
     "Parcel Collected",
     "Shipping Labels Downloaded",
+    "Driver Overview Downloaded",
```

Comment:
Could you create a corresponding migration file to add those statuses to `status_order` please?

Reply:
should i edit the existing migration file "add_website_data_and_status_orders.sql" directly?

Reply:
Create a new migration file please.

If we add it to that migration file which has already been applied on deployed db, then the migration won't run when we deploy the new version because supabase will think it has already run that migration.

Reply:
Should I do this by writing the SQL myself?

Reply:
Yes please! I'd assume it's similar to what's in https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/blob/dev/supabase/migrations/20240409135335_add_website_data_and_status_orders.sql

Reply:
Thanks!